docs/user/accumulo/data_management.rst

```diff
@@ -42,6 +42,129 @@ be answered without joining against the record table. This is the only option fo
 To use a full index, the keyword ``full`` must be used in place of ``true`` when specifying an attribute
 index in the ``SimpleFeatureType``.  Note that join indices are the default in the Accumulo data store.
 
+Accumulo Logical Timestamps
+---------------------------
+
+By default, GeoMesa index tables are created using Accumulo's logical time. This ensures that updates to a given
+simple feature will be ordered correctly, however it obscures the actual insert time for the underlying data
+row. For advanced use cases, standard system time can be used instead of logical time. To disble logical
+time, add the following user data hint to the simple feature type before calling ``createSchema``:
+
+.. code-block:: java
+
+    // append the hints to the end of the string, separated by a semi-colon
+    String spec = "name:String,dtg:Date,*geom:Point:srid=4326;geomesa.logical.time='false'";
+    SimpleFeatureType sft = SimpleFeatureTypes.createType("mySft", spec);
+
+.. warning::
+
+    If table sharing is enabled, then the index tables may have already been created with logical
+    time enabled. If this is the case, then disabling it will have no effect. To ensure the
+    tables are created correctly, you may wish to disable table sharing by adding the user data string
+    ``geomesa.table.sharing='false'``
+
+Splitting the Record Index
+--------------------------
+
+By default, GeoMesa assumes that feature IDs are UUIDs, and have an even distribution. If your
+feature IDs do not follow this pattern, you may define a custom table splitter for the record index.
+This will ensure that your features are spread across several different tablet servers, speeding
+up ingestion and queries.
+
+GeoMesa supplies three different table splitter options:
+
+- ``org.locationtech.geomesa.index.conf.HexSplitter`` (used by default)
+
+  Assumes an even distribution of IDs starting with 0-9, a-f, A-F
+
+- ``org.locationtech.geomesa.index.conf.AlphaNumericSplitter``
+
+  Assumes an even distribution of IDs starting with 0-9, a-z, A-Z
+
+- ``org.locationtech.geomesa.index.conf.DigitSplitter``
+
+  Assumes an even distribution of IDs starting with numeric values, which are specified as options
+
+Custom splitters may also be used - any class that extends ``org.locationtech.geomesa.index.conf.TableSplitter``.
+
+Specifying a Table Splitter
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Table splitter may be specified by setting a hint when creating a simple feature type,
```

Comment:
Ok, what does a table splitter do?  This section could have a one or two about why I'd consider using one.

Reply:
see other comment, i didn't actually modify this part of the docs (although your question is reasonable :)

Reply:
push it to a new ticket if we want a doc review for next time. this is just a move